src/main/kotlin/de/bund/digitalservice/useid/refresh/RefreshController.kt

```diff
@@ -0,0 +1,38 @@
+package de.bund.digitalservice.useid.refresh
+
+import de.bund.digitalservice.useid.identification.IdentificationSessionService
+import mu.KotlinLogging
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+import java.net.URI
+import java.util.UUID
+
+@RestController
+@RequestMapping("/refresh")
+class RefreshController(private val identificationSessionService: IdentificationSessionService) {
+
+    private val log = KotlinLogging.logger {}
+
+    @GetMapping
+    fun redirectService(@RequestParam sessionId: UUID): Mono<ResponseEntity<Unit>> {
+        return identificationSessionService.findByEIDSessionId(sessionId)
+            .doOnError { exception ->
+                log.error { "error occurred while checking $sessionId;\n ${exception.message}" }
+            }
```

Comment:
Even if it's a little longer that way, I would make the naming a little bit more explicit. And please log the whole exception, not only the exception message (by providing it to the log message as last parameter). Otherwise the stack trace will not be logged.  

```suggestion
    fun redirectToEServiceRefreshAddress(@RequestParam("sessionId") eIDSessionId: UUID): Mono<ResponseEntity<Unit>> {
        return identificationSessionService.findByEIDSessionId(eIDSessionId)
            .doOnError {
                log.error("Failed to load identification session with eIDSessionId: $eIDSessionId", it)
            }
```